Documentation/data-format.md

```diff
@@ -0,0 +1,69 @@
+# Adobe Experience Platform Edge Bridge for iOS
+
+## Data Format
+
+The AEP Edge Bridge extension maps the data sent in `trackAction` and `trackState` API calls to a specific format known to Analytics which requires no additional server-side mapping. This document outlines the data format applied to the tracking data.
+
+
+As of version 5.0.0 of AEP Edge Bridge for iOS, the following table lists the mapping of the `trackAction` and `trackState` parameters to the "data" element of the Experience Event sent to the Edge Network.
+
+
+| Data | Key in dispatched request | Comments |
+| --- | --- | ---
+| action | `data.__adobe.analytics.linkName` | Additionally, the field `data.__adobe.analytics.linkType` with value `lnk_o` is automatically included. |
+| state | `data.__adobe.analytics.pageName` | |
+| contextdata | `data.__adobe.analytics.contextData` | Context data is a map which includes the custom keys and values specified in the `trackAction` and `trackState` API calls. |
+| prefixed contextdata | `data.__adobe.analytics` | Context data keys prefixed with `&&` must be known to Analytics and are case sensitive. When mapped to the event, the key's name does not include the "&&" prefix.|
```

Comment:
Data column: `&& prefixed context data.`
Description column: For example &&products is sent as data.__adobe.analytics.products.